ui_framework/components/_common_styles.scss

```diff
@@ -15,3 +15,10 @@
 body {
   font-family: $globalFontFamily;
 }
+
+/**
+ * 1. Required for IE11.
+ */
+main {
+  display: block; /* 1 */
+}
```

Comment:
Ah I think I should probably rename that task. `npm run uiFramework:start` will recompile the compiled CSS, whereas `uiFramework:build` will only build the documentation site.